.gitignore

```diff
@@ -55,4 +55,7 @@ docs/_build/
 .cache
 
 .*
-archive
\ No newline at end of file
+archive
+
+# Main 
+main.py
```

Comment:
I feel like this was added for your own project, but this is not necessary in our repository.
```suggestion
```